codecov.yml

```diff
@@ -2,11 +2,17 @@ coverage:
   status:
     project:
       default:
-        target: 90%
+        target: auto
         threshold: 1%
+        base: auto
 
 comment:
   layout: 'header, reach, diff, flags, components'
+  behavior: default
+  require_changes: false
```

Comment:
> new baseline is uploaded correctly, but why don't we see the CI check here or a comment from the bot?

`comment.require_changes: false` should force make comment from the bot without changes

https://docs.codecov.com/docs/quick-start#tips-and-tricks
<img width="835" alt="image" src="https://github.com/TanStack/query/assets/61593290/a608181e-06cb-4572-a725-9ecd76be4b65">